spec/index.html

```diff
@@ -791,6 +791,16 @@ <h3>Version Labels</h3>
       </tbody>
     </table>
 
+    <p>
+      Data conforming to [[[RDF11-MT]]] has the same semantics as given by [[[RDF12-SEMANTICS]]].
+    </p>
```

Comment:
```suggestion
      Data conforming to version "1.1" has the same semantics under [[[RDF11-MT]]] and under [[[RDF12-SEMANTICS]]].
    </p>
```

I find this wording clearer. It would even better if this sentence appeared after the other one (pointing out that data conforming to version "1.1" also conforms to "1.2-basic" and "1.2", which explains why we consider its semantics under RDF12-SEMANTICS...